near-sdk/src/store/lookup_map/impls.rs

```diff
@@ -25,7 +26,8 @@ impl<K, V, H, Q: ?Sized> core::ops::Index<&Q> for LookupMap<K, V, H>
 where
     K: BorshSerialize + Ord + Clone + Borrow<Q>,
```

Comment:
I don't think we actually need `Clone` here so we can relax the constraint

Reply:
yup! That was an artifact of a previous API iteration, although I don't think restrictive in any sound implementation. Removed :)